dojo/tools/twistlock/parser.py

```diff
@@ -95,6 +100,38 @@ def parse_issue(self, row, test):
 
         return finding
 
+    def get_description(data_description,
+                        data_type,
+                        data_tag,
+                        data_cluster,
+                        data_namespaces,
+                        data_package_name,
+                        data_package_version,
+                        data_package_path,
+                        data_name,
```

Comment:
the parameters data_name, data_cloud_id, data_runtime, data_cause, data_found_in and data_purl are not used.